### ISSUE TITLE
Up food value of cured meat

### ISSUE THREAD

**Issue by LannyRipple** [BTW-Community/BTW-Public#283]
Cured meat is basically useless at the moment.  Early game you have no nitre.  By the time you have a mob farm and enough food to stack 64 cured meat you can simply take 2 stacks of the food you used to get the stack.  (You might also have milk and/or wheat by this time meaning you wouldn't waste the food with curing.)

Up the value of cured meat to 1.5 shanks making it appealing for early game and later inventory slot-saving use.

**Comment by LannyRipple** [comment]
I know there has been some discussion of this already so if there is already a plan for cured meat this issue can be closed.

**Comment by ammoore00** [comment]
Open to the idea. As long as it stays below the 2 shanks of cooked meat i think it would still fit into the same gameplay niche.

Because yeah as is i don't think I've ever once used cured meat.

**Comment by PlasmaFox256** [comment]
Cured meat is supposed to be an emergency food when you are starving in the early game, not a midgame lazy food. I'd say if you increase its hunger replenish, you should decrease the stack size back to 16. Cooking and crafting better meals once you have the infrastructure to do so was FC's idea and I feel like cured meat was made inefficient on purpose.

**Comment by LannyRipple** [comment]
The odds of having nitre in the early game and not being able to make a
campfire seem awfully low.


   - Added Cured Meat as a new food item. It is created by combining a
   piece of raw meat (or fish or chicken), and Nitre in the crafting grid.
   Curing meat actually decreases the hunger value of raw food items, but it
   does eliminate any chance of food poisoning, and cured meats may also be
   stacked in greater amounts for easier storage. Note that eating too many
   cured meats may result in a mild BURNing sensation.


A food value of 1.5 is still sub-standard (2 for cooked chicken or fish)
and captures the idea of stacking greater amounts for easier storage.

**Comment by Hirachosan** [comment]
With Fatty Foods addon I upped the value as well. I think it's important that it is reduced food value compared to cooked. so that it fills a niche. Basically what Lanny said Is what I had and what I think works best. FC made them stack to 32 IIRC. it basically gives it a little niche use. As I discussed before on Discord.. right now it is in a weird spot because youd only ever make it in a situation where its almost impossible or counterintuitive to get nitre. that being super early game where killing creepers means making them explode :P
